### DISCUSSION TITLE
"Debugger not found", even though everything has been installed correctly

### DISCUSSION THREAD

**Discussion by confusedcatgirl** [CosmosOS/Cosmos#2870] [Q&A]
**Exception**
See Section: Screenshots

**How To Reproduce**
1. Install Cosmos Dev Kit
2. Build it (Obviously)
3. Create a new Project
4. Build the Project

**Screenshots**
![image](https://github.com/CosmosOS/Cosmos/assets/64417934/350710a0-c8a4-475c-8310-29d53f9e244d)

**Context**
Before posting please confirm that the following are in order
[x] Both Cosmos VS Extensions are installed
[x] In the NuGet Package Manager "Include prerelease" is selected
[x] The Cosmos NuGet package store is selected (NOT nuget.org) in 'Manage NuGet Packages'
[x] The Cosmos NuGet packages are installed

**Context**

I use the newest Version of the Cosmos Dev Kit, built today. I have installed all prerequisites, and did everything necessary to install cosmos on a windows 10 machine. Now, when I try to run a project, it builds the .iso, but never runs any program I have installed, and just throws the error seen above. I have installed everything several times over, and I am starting to question my own sanity. Not Hyper-V works, VMWare doesn't work, I added both of them into my PATH and I am slowly giving up.
It would be absolutely incredible if you could help me with my problem.

**Comment by quajak** [comment]
Does Bochs work?
